src/angularjs/src/app/places/detail/place-map.directive.js

```diff
@@ -179,25 +236,44 @@
             function buildLabel(properties) {
 
                 // omit some less-useful properties
-                var ignore = ['blockid10', 'osm_id'];
+                var ignore = ['blockid10', 'osm_id', 'fatality_type'];
+                var fatality_type = properties['fatality_type']
+                var title = ''
+                if (properties['fatality_type']) {
+                    fatality_type = fatality_type.toLowerCase().replace(/_/g, ' ').replace(/active/g,'active transportation')
+                    title = 'Fatal crash'
+                }
                 properties = _.omitBy(properties, function(val, key) {
                     return _.find(ignore, function(i) {return i === key;});
                 });
 
-                var snippet = '<ul>';
+                var snippet = '<h3 class="leaflet-popup-content-header">'+title+'</h3>';
                 snippet += _.map(properties, function(val, key) {
+                    var unjoinedHtml = []
+                    var label = key.replace(/fatality_count/g, 'fatalities').replace(/_/g, ' ')
+                    label = label.charAt(0).toUpperCase() + label.slice(1)
+                    unjoinedHtml.push('<div class="leaflet-popup-content-row">')
+                    unjoinedHtml.push('<div class="leaflet-popup-content-key">'+label+'</div>')
+
                     // humanize numbers: round to 3 digits and add commas
-                    if (Number.parseFloat(val)) {
+                    if (key !== 'year' && Number.parseFloat(val)) {
                         val = $filter('number')(val);
                     }
-                    return ['<li>',
-                            key.replace(/_/g, ' '),
-                            ': ',
-                            (val ? val : '--'),
-                            '</li>'
-                            ].join('');
-                }).join('');
-                snippet += '</ul>';
+
+                    unjoinedHtml.push('<div class="leaflet-popup-content-val">'+(val ? val : '--')+'</div>')     
+                    unjoinedHtml.push('</div>')
+                
+                    if (label === 'Fatalities' && val > 1 && fatality_type !== 'motor vehicle') {
+                        unjoinedHtml.push('<div class="leaflet-popup-content-row-hint">Involving at least 1 ' + fatality_type + ' fatality'+'</div>')
```

Comment:
the logic just changed to combine this with the message for active transportation, but basically:
- any time we have a crash with active transportation fatalities, we need to be clear about what that means
- any time we have multiple fatalities in the active transportation or bike categories, we need to be clear the number of fatalities might include other types of fatalities. this isn't the case for motor vehicles, because if there's at least 1 bike, the fatality type will always be bike + if there's 1 or more active type fatality but no bike fatalities, the fatality will always be active.

I will add a clarifying comment lol

Reply:
Ok, cool.  I think I just hadn't managed to click on the right category of marker.  (I tweaked the filter in the view to make it easier to find one.)